packages/create-vite/template-svelte-ts/src/style.css

```diff
@@ -0,0 +1,46 @@
+:root {
```

Comment:
It might be better to be `index.css`?

Reply:
@benmccann what is more idiomatic in Svelte? `index.css` or `style.css`?

Reply:
We use `app.css` in SvelteKit :grin:

Reply:
updated, thanks!